modules/datastore/src/PostImportResult.php

```diff
@@ -51,37 +50,86 @@ class PostImportResult {
    */
   protected ResourceMapper $resourceMapper;
 
-  /**
-   * The PostImport service.
-   */
-  protected PostImport $postImport;
-
   /**
    * The PostImportResult.
    */
   protected array $postImportResult;
 
   /**
-   * PostImportResource constructor.
+   * PostImportResult constructor.
    */
   public function __construct(
     $postImportResult,
+    Connection $connection,
     ResourceMapper $resourceMapper,
```

Comment:
After some discussion we don't believe the resourceMapper is necessary here. Confirm, and then remove this plus the code that uses it to find the latest version ID.